src/main/CommandHandler.cpp

```diff
@@ -534,12 +535,13 @@ CommandHandler::catchup(std::string const& params, std::string& retStr)
         }
     }
 
-    mApp.getLedgerManager().startCatchUp(ledger, mode, true);
+    mApp.getLedgerManager().startCatchUp(ledger, count,
+        VerifyLedgerMode::DO_NOT_VERIFY_BUFFERED_LEDGERS);
     retStr = (std::string("Started catchup to ledger ") +
               std::to_string(ledger) + std::string(" in mode ") +
-              std::string(mode == CatchupManager::CATCHUP_COMPLETE
+              std::string(count == std::numeric_limits<uint32_t>::max()
                               ? "CATCHUP_COMPLETE"
-                              : (mode == CatchupManager::CATCHUP_RECENT
+                              : (count > 0
```

Comment:
`count != 0` would be more accurate as it's an `uint32_t`

Reply:
ok